lang/po/Firmware_cs.po

```diff
@@ -2573,6 +2573,11 @@ msgstr "Připravit"
 msgid "Set not Ready"
 msgstr "Zrušit Připravena"
 
+#. MSG_HOSTPRINT c=18
+#: ../../Firmware/messages.cpp:107 ../../Firmware/ultralcd.cpp:5285
+msgid "Print from host"
+msgstr "Tisk z hostitele"
```

Comment:
Same applies to RO, but I can't think of another word to refer to the printer host.

Reply:
In that case "Tisk z hosta", even though it doesn't make sense to non-technical people.
May be even "Tisk z RPi" would make sense

Reply:
Hmmm that is still not ideal. In other places in the fw where we say RPi we strictly mean the UART1 port on the back of the einsy, but here it could be USB as well and not limited to a raspberry pi. "Print from USB" is also not a good replacement as it has the same issues, but in reverse where USB doesn't make sense for the rpi port.